controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3682,7 +3682,7 @@ OperationFailedRuntimeException capabilityAlreadyRegisteredInContext(String capa
     IllegalArgumentException noResourceForUndefiningAttribute(String attribute, String address);
 
     @LogMessage(level = WARN)
-    @Message(id = 490, value = "You have defined a resource for address %s without any attributes, doing nothing")
+    @Message(id = 490, value = " A YAML resource has been defined for the address %s without any attribute. No actions will be taken.")
```

Comment:
I guess this white space at the beginning is not intentional for this message that contains an ID, could you confirm @ehsavoie

Reply:
correct